docs/new-architecture-backward-turbomodule.md

```diff
@@ -0,0 +1,699 @@
+
+```sh
+npx create-react-native-library <name-of-the-library>
+```
+
+Follow the questions the CLI asks and, once the process is done, it will create a folder structure completed with an example app, typescript configuration and much more. The sample structure is shown in the following image:
+
+<figure>
+  <img src="docs/assets/Turbomodules/NpxLibFolderStructure.png" />
+  <figcaption>The folder structure generated by the React Native CLI</figcaption>
+</figure>
+
+This is a good starting point, but remember to double check what will be published if you keep this setup. Moreover, the `TypeScript` support for the new architecture is still in beta, so you won't be able to use it for the Turbomodule sections.
+
+#### Manually
+
+The second approach is to create a similar structure manually. What we need to start is a folder named `example-library` and a `package.json` file with the following content:
+
+```json title="package.json"
+{
+  "name": "example-library",
+  "version": "0.0.1",
+  "description": "Showcase Turbomodule with Backcompatibility ",
+  "react-native": "src/index",
+  "source": "src/index",
+  "files": [
+    "src",
+    "android",
+    "ios",
+    "example-library.podspec",
+    "!android/build",
+    "!ios/build",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
+  ],
+  "keywords": ["react-native", "ios", "android"],
+  "repository": "https://github.com/<your_github_handle>/example-library",
+  "author": "<Your Name> <your_email@your_provide.com> (https://github.com/<your_github_handle>)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/<your_github_handle>/example-library/issues"
+  },
+  "homepage": "https://github.com/<your_github_handle>/example-library#readme",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "devDependencies": {},
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  }
+}
+```
+
+For sake of simplicity, we will go on with this manual setup, adding files while we work.
+Remember to update the `<Your Name>`, `<your_email@your_provide.com>`, and all the `<your_github_handle>` tokens, if you want to follow this approach with your information.
+
+### Create a Testing Environment
+
+To develop a Native Module we would like to test the JS side and to develop safely the native parts. A library usually does not come with a testing aplication out-of-the-box. So, we are going to need one.
+
+:::note
+If you used the React Native CLI to create the module, you have an application called `example` within you `example-library` folder.
+:::
+
+To create a proper test environment, let's navigate to the parent folder of our `example-library` and run the following commands:
+
+```sh
+npx react-native init OldArchitectureApp
+cd OldArchitectureApp
+yarn install
+```
+
+Now, open a new terminal and start Metro by running:
+
+```sh
+npx react-native start
+```
+
+Go back to previous terminal and run:
+
+```sh
+npx react-native run-ios
+```
```

Comment:
```suggestion
```sh
npx react-native run-ios
# Or for Android
npx react-native run-android
```
```

Also some of those basic commands can be omitted and we can assume a basic knowledge of the RN CLI